privaterelay/tests/mgmt_sync_phone_related_dates_on_profile_tests.py

```diff
@@ -85,13 +86,12 @@ def test_free_phone_user_gets_only_date_phone_subscription_reset_field_updated(
 ):
     expected_now = patch_datetime_now
     mocked_dates.return_value = (None, None, None)
-    account = baker.make(SocialAccount, provider="fxa")
+    account: SocialAccount = baker.make(SocialAccount, provider="fxa")
     profile = Profile.objects.get(user=account.user)
     baker.make(Flag, name="free_phones")
-    free_phones_flag = Flag.objects.filter(name="free_phones").first()
+    free_phones_flag = Flag.objects.get(name="free_phones")
     free_phones_flag.users.add(profile.user)
     free_phones_flag.save()
```

Comment:
suggestion (non-blocking): this block is used at least twice. Can we move this logic into a `get_user_free_phones()` helper function?